voting/client-web/src/components/Header/Header.tsx

```diff
@@ -29,6 +30,20 @@ const HeaderImpl = ({ classes, isReadOnly }) => {
     { label: t('Rewards'), url: '/reward' },
   ];
 
+  function displayInMetamask() {
+    (window as any).web3.currentProvider.sendAsync({
+      method: 'metamask_watchAsset',
+      params: {
+        type: 'ERC20',
+        options: {
+          symbol: 'ORBS',
+          address: '0xff56cc6b1e6ded347aa0b7676c85ab0b3d08b0fa',
+          decimals: 18,
+          image: 'https://s2.coinmarketcap.com/static/img/coins/64x64/3835.png',
```

Comment:
Coded hard ?

Reply:
If you're talking about the image url, I think that it's self explanatory.